### ISSUE TITLE
[feature] embeddable Hoppscotch Instances

### ISSUE THREAD

**Issue by abishekram** [hoppscotch/hoppscotch#1831]
Is it possible to consume as standalone Vue Component ?

Is there any idea on releasing as NPM package ?. If that's possible, we could use hoppscotch like Swagger-UI which could be used all in servers. So the API contributors could expose their API usage through hoppscotch.

Hoppscotch could fill up that space and can help in API development.

**Comment by liyasthomas** [comment]
Embeddable Hoppscotch instances are in our roadmap. Watch this space for updates. Thank you.

**Comment by buger** [comment]
👍🏻

**Comment by juzhiyuan** [comment]
Hi @liyasthomas, may I know where could I find the Roadmap? 😄

**Comment by liyasthomas** [comment]
We have a private roadmap. Hopefully, we'll be able to publish it publically one day.

**Comment by unredundant** [comment]
Hi, is there any progress / timeline on this feature? 

My use case is that I would like to embed a collection as part of a service catalog entry inside of a [Backstage](https://backstage.io/) instance.  To do so, I would need to build a Backstage plugin that features an embedded collection :)

**Comment by liyasthomas** [comment]
We've introduced a feature that allows users to embed Hoppscotch's RESTful API playground in any HTML document via an iframe. Consider this as a starting point in making Hoppscoth accessible for API consumption by general users.

![Frame 7](https://github.com/hoppscotch/hoppscotch/assets/10395817/a3ea27b1-c059-4238-9bcc-21b09f8ee3d6)

Thanks for your patience and valuable feedback. This feature has been implemented in the latest release.

Closing this ticket as this feature is now available in the recent version. Feel free to reach out if you encounter any further concerns.

Blog: https://hoppscotch.com/blog/better-api-testing-capabilities-and-more
Docs: https://docs.hoppscotch.io/documentation/features/widgets#embeds
